src/module.ts

```diff
@@ -18,6 +18,7 @@ import { listen } from 'listhen'
 import type { WatchEvent } from 'unstorage'
 import { name, version } from '../package.json'
```

Comment:
```suggestion
import { name } from '../package.json'
```

Reply:
Version is used in package `meta` https://github.com/nuxt/content/blob/feat/internal-cache-version/src/module.ts#L173